### PR TITLE
Retrieve and reuse autopilot recipes

### PR DESCRIPTION
**Pull request by Changshama**
Train autopilot models using subset of data and reuse its recipes on the whole dataset

*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: autopilot/ap-batch-transform.ipynb [hunk 1]
```diff
@@ -0,0 +1,1297 @@
+{
```
Comment (atqy) [inline autopilot/ap-batch-transform.ipynb:1]: <p>Need to add badges. Just follow the instructions under "Add CI badges to your notebook" in&nbsp;<a href="https://github.com/aws/amazon-sagemaker-examples/blob/main/CONTRIBUTING.md" target="_blank">https://github.com/aws/amazon-sagemaker-examples/blob/main/CONTRIBUTING.md</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (atqy) [inline autopilot/ap-batch-transform.ipynb:1]: <p>Download the file from s3://sagemaker-example-files-prod-REGION/datasets/tabular/uci_bank_marketing/bank-additional-full.csv</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (atqy) [inline autopilot/ap-batch-transform.ipynb:1]: <p>Clear outputs to keep notebook clean andavoid revealing sensitive data like account numbers, etc. Do this for all cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (atqy) [inline autopilot/ap-batch-transform.ipynb:1]: <p>Not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (Changshama) [reply autopilot/ap-batch-transform.ipynb:1]: added

**Pull request merged**